src/vmshepherd_aws_drivers/aws_iaas_driver.py

```diff
@@ -0,0 +1,84 @@
+import asyncio
+import aiobotocore
+from typing import Any, Dict, List
+from vmshepherd.iaas import AbstractIaasDriver, Vm, VmState
+
+
+class AwsIaaSDriver(AbstractIaasDriver):
+
+    async def list_vms(self, preset_name):
+        '''
+        List VMs by preset name
+        :arg present_name: string
+        '''
+        loop = asyncio.get_event_loop()
+        session = aiobotocore.get_session(loop=loop)
+        async with session.create_client('ec2') as client:
+            res = await client.describe_instances(
+                Filters=[
+                    {
+                        'Name': 'tag:aws:autoscaling:groupName',
+                        'Values': [preset_name]
+                    }
+                ]
+            )
+            instances = []
+            for vms in res['Reservations']:
+                instances.extend(vms['Instances'])
+        result = []
+        for instance in instances:
+            result.append(self._map_vm_structure(instance))
+        return result
+
+    async def create_vm(self, preset_name: str, image: str, flavor: str, security_groups: List=None,
```

Comment:
SInce non of these parameters is being used, how about 
```suggestion
    async def create_vm(*args, **kwargs)
```